assets/.babelrc

```diff
@@ -2,4 +2,4 @@
     "presets": [
         "@babel/preset-env"
     ]
-}
+}
```

Comment:
Is it intentional to have no newlines at the end of all files?

Reply:
It isn't. I bet my editor did that. I'll correct

Reply:
This should be fixed now. I added an eslint rule to catch it for the js files in the future, I don't think it will catch on this config files though.